File: src/hasoftware-library/src/hasoftware/util/EventType.java
package hasoftware.util;

public enum EventType {

    None(0, "None"),
    TimeCheck(1, "Timecheck"),
    Shutdown(2, "ShutDown"),
    Connect(3, "Connect"),
    Disconnect(4, "Disconnect"),
    ClientConnect(5, "Client Connect"),
    ClientDisconnect(6, "Client Disconnect"),
    SendMessage(7, "Send Message"),
    ReceiveMessage(8, "Recveive Message");

    private final int _id;
    private final String _code;

    EventType(int id, String code) {
        _id = id;
        _code = code;
    }

    public int getId() {
        return _id;
    }

    public String getCode() {
        return _code;
    }
}

File: src.net/hasoftware-library/Util/EventType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace hasoftware.Util
{
    public enum EventType
    {
        [Description("None")]
        None = 0,
        [Description("Timecheck")]
        TimeCheck = 1,
        [Description("Shutdown")]
        Shutdown = 2,
        [Description("Connect")]
        Connect = 3,
        [Description("Disconnect")]
        Disconnect = 4,
        [Description("ClientConnect")]
        ClientConnect = 5,
        [Description("ClientDisconnect")]
        ClientDisconnect = 6,
        [Description("SendMessage")]
        SendMessage = 7,
        [Description("ReceiveMessage")]
        ReceiveMessage = 8
    }
}
